windows.Dockerfile

```diff
@@ -0,0 +1,99 @@
+# escape=`
+
+# ------------------------------------------------------------------------
+#
+# Copyright 2020 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/dotnet/framework/sdk:4.8
+
+SHELL ["powershell.exe", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# See https://github.com/jenkinsci/docker-agent/blob/master/8/windows/windowsservercore-1809/Dockerfile
+
+ENV JAVA_VERSION jdk8u252-b09
+
+USER ContainerAdministrator
```

Comment:
@gayashanbc just a suggestion. Maybe we can add a short comment indicating why we are switching users along the way.

Reply:
That switch was redundant and hence removed in https://github.com/wso2/build-docker/pull/18/commits/8bd6ae87c078b011e391265c6f7a479d59c8b05e